### ISSUE TITLE
at_c: Build examples CI

### ISSUE THREAD

**Issue by JeremyTubongbanua** [atsign-foundation/at_c#48]
As atclient and atchops matures, some CI to check that the examples still build would be nice to have.

**Comment by JeremyTubongbanua** [comment]
Assigning 1 SP;

Work involved is 1 GitHub workflow to loop through each folder in `examples/` to run (similar to) the following:
```sh
cd <folder>
cmake -S . -B build
cmake --build build
cd build && ./<target_executable>
```

P.S.: build the atclient (and atchops) library beforehand:
```sh
cd packages/atclient
... (see atclient/README.md or other CI on how to build the atclient source)
```

**Comment by JeremyTubongbanua** [comment]
Closed by #214
